src/table/sticky-scrollbar/sticky-scrollbar.tsx

```diff
@@ -39,7 +46,7 @@ function StickyScrollbar(
       ref={mergedRef}
       className={clsx(
         styles['sticky-scrollbar'],
-        offsetScrollbar && styles['sticky-scrollbar-offset'],
+        (offsetScrollbar || forceOffsetScrollbar) && styles['sticky-scrollbar-offset'],
```

Comment:
If we do this we won't need the `offsetScrollbar` argument, because it will cover all cases.

Reply:
> Looks like sticky scrollbar always enables the offset regardless of the scrollbar style.
I'm not sure I understood this.

> Was the overlap of the scrollbar over interactive content the only reason we did this? 
Yes!